README.md

```diff
@@ -82,3 +82,19 @@ A Repository to collect and showcases developers' favorite resources to learn a
     - **What didn't I understand?** Great introduction to client/server side code but definitely more pracitce needed afterwards.
     
   ---
+### [Gayathri Krishnan](https://github.com/gayathri-krishnan)  
+> Working mom of two kids, Java/J2EE developer with sound DB knowledge. Worked across technologies over the years. Find me on [LinkedIn](https://www.linkedin.com/in/gayathri-krishnan-24738148/)
+
+  1. [Algorithms - Coursera Algorithmic toolbox](https://www.coursera.org/learn/algorithmic-toolbox)
+     - **Why I like it?** Very well structured and explained. Starts from O(n), works through different algorithms. 
+     - **What worked for me?** The examples and assignments
+     - **What didn't I understand?** Why some of my code failed due to execution time. But that's me. I would highly recommend
+    
+  2. [Udemy](https://udemy.com)
```

Comment:
@gayathri-krishnan Thank you so much for contributing your favorite resources

Few things I would like to add "medium" and "udemy" are too broad. We want people to add only the courses/blogs/resources they used, tried and worked with and not platforms.

If you have specific udemy courses and medium blogs feel free to add them